WordPress/src/main/res/values/strings.xml

```diff
@@ -1370,7 +1370,7 @@
     <string name="delete_site_hint">Delete your site</string>
     <string name="pref_key_site_start_over_screen">wp_pref_key_site_start_over_screen</string>
     <string name="delete_site_progress">Deleting site…</string>
-    <string name="purchases_request_error">Something went wrong. Please contact support</string>
+    <string name="purchases_request_error">We could not request purchases. Please contact support</string>
```

Comment:
Using "we" isn't recommended - see "Don't refer to we" on [this page](https://www.google.com/design/spec/style/writing.html#writing-language). Also, we should drop "Please contact support" since this is shown as a toast and provides no way for the user to contact support.